test-app/tests/integration/components/bs-dropdown-test.js

```diff
@@ -50,35 +50,14 @@ module('Integration | Component | bs-dropdown', function (hooks) {
     assert.dom(dropleftClass).exists('has dropleft class');
   });
 
-  test('dropdown container with dropdown button has btn-group class', async function (assert) {
+  test('supports setting btn-group class', async function (assert) {
     await render(
-      hbs`<BsDropdown as |dd|><dd.button>Dropdown
-    <span class='caret'></span></dd.button><dd.menu><li><a
-        href='#'
-      >Something</a></li></dd.menu></BsDropdown>`,
-    );
-    assert.dom('.btn-group').exists('has btn-group class');
-  });
-
-  test('dropdown container with block dropdown button has dropdown class', async function (assert) {
```

Comment:
Is there a reason for removing the dropdown and dropup tests? According to the jsdoc of `bs-dropdown containerClass`, they should still be supported

Reply:
The tests were asserting that `.btn-group` CSS class is used in some cases instead of `.dropdown`. That logic is removed in the PR. It is now using `.dropdown` CSS class only. `.btn-group` is not used in any case anymore.

There is another test (`dropdown container has dropdown class`), which covers that the markup is as expected.